test/check-cli

```diff
@@ -102,6 +102,20 @@ class TestTar(composertest.ComposerTestCase):
             # Run the test, on the booted image
             self.runImageTest("/tests/cli/test_boot_tar_kickstart.sh")
 
+class TestPylint(composertest.ComposerTestCase):
+    def test_ci(self):
+
+        # Create an empty folder for the upload bellow
+        self.execute(["mkdir /lorax"])
+
+        # Upload contents of repo for running the test
+        self.machine.upload(["../tests"], "/lorax/tests")
+        self.machine.upload(["../src"], "/lorax/src")
+        self.machine.upload(["../share"], "/lorax/share")
+        self.machine.upload(["../Makefile"], "/lorax")
```

Comment:
@bcl what do you think about this ? Also see comment in 
https://github.com/weldr/lorax/pull/882#discussion_r343122821

It's not particularly elegant as would be copying the entire directory or even better bind-mount the git checkout into the running VM. However it seems to work.

Reply:
That's fairly awkward. But bind mounting would be worse, allowing changes to the host. Copying all of ./lorax/ would also include any .git tree and extra artifacts, so I guess this is the least bad option.
upload can take a list of things to upload, you should be able to do this instead:
`self.machine.upload([../tests", "../src", "../share", "../Makefile"], "/lorax")`

Reply:
Yes it is awkward. I changed it to the list now.